src/Agent.Service/Windows/AgentService.cs

```diff
@@ -151,7 +151,7 @@ private Process CreateAgentListener()
             string exeLocation = Assembly.GetEntryAssembly().Location;
             string agentExeLocation = Path.Combine(Path.GetDirectoryName(exeLocation), "Agent.Listener.exe");
             Process newProcess = new Process();
-            newProcess.StartInfo = new ProcessStartInfo(agentExeLocation, "run");
+            newProcess.StartInfo = new ProcessStartInfo(agentExeLocation, "run --startuptype windowsservice");
```

Comment:
wasnt aware of this usage till now. 

---
In reply to: [122278125](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r122278125) [](ancestors = 122278125)